dbms/src/Flash/EstablishCall.h

```diff
@@ -95,6 +96,8 @@ class EstablishCallData final
     grpc::ServerContext * getGrpcContext() { return &ctx; }
 
     String getResourceGroupName() const { return resource_group_name; }
+    grpc::Alarm * getAlarm();
```

Comment:
Better to use a reference instead of a pointer because this pointer is always not null.

Reply:
Can't use reference here because we need to store it in a std::unordered_map in `MPPTaskManager`

Reply:
`std::reference_wrapper` can be used to store a reference in `unordered_map`

Reply:
Done